### PR TITLE
PR for the trial-branch

### PR DESCRIPTION
**Pull request by KALUNGI-J-BOBSON**
added the html
setup linters

### CODE REVIEW

**Review by KALUNGI-J-BOBSON — COMMENTED** [review]
thats good but i think you were supposed to have created the main branch first
Linters Pass
Good github flow

**Review by KALUNGI-J-BOBSON — COMMENTED** [review]
Linters pass 
But iu think you would have to update your readme.md file and give a brief desrciption about you project

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @KALUNGI-J-BOBSON, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Polite tone was used ✅
## Required Changes ♻️
- To be able to review your code effectively, kindly click on the pull request, and after that click on file changes 
image

- From the file changes section is where you will be able to leave comments on the code that needs changes 😊🟢
- You can follow this [link ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)on how to give a code review ⚔️🟢
- Kindly tell the programmer why you are requesting changes, this will enable them to know why they are doing what you asked them to do😎😊

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- I think it will be nice if you make use of emojis and gifs, this will make you appear friendly to the programmer 😎😉
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KALUNGI-J-BOBSON — COMMENTED** [review]
Hello 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Highlights
Linter Pass ✅
good Git hub flow ✅
Required  Changes ♻️
Kindly make a more elaborative README.md 
I think it would be better if you included future features about you project and FAQs you think someone may ask about and providing briefs answers 😎😉

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by Debas-31 — CHANGES_REQUESTED** [review]
> ## Hi @KALUNGI-J-BOBSON  :wave:, 
# Status: Require Change :recycle:
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights
- No linters errors :heavy_check_mark: 
- Good README file :heavy_check_mark: 
 ## Required Changes ♻️
- [ ] Please kindly use inline comments on your Microverse trial projects or previous HTML and CSS projects. To review inline comments, follow the procedure of previous code reviewer suggestions and [how to code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features).

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -9,7 +9,7 @@ body {
 #main {
   margin: 30px 8px;
   border-radius: 5px;
-  padding: 15px;
```
Comment (KALUNGI-J-BOBSON) [inline style.css:12]: Hello, looks good so far no issues just to merge your branch

**Review by KALUNGI-J-BOBSON — COMMENTED** [review]
Your project is complete Your project is complete! There is nothing else to say other than... it's time to merge it  
Congratulations! 🎉
Highlights
•	Good commit history ✔️
•      Linters are passing ✔️

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @KALUNGI-J-BOBSON, 

Your project is complete! Great job for following the project requirement🥇and implementing changes requested by the previous reviewers. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- Proper use of review features ✔️
- proper use of in-line comment ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**